### PR TITLE
Deprecations 1.1

### PR DESCRIPTION
**Pull request by drammock**
Let's see what I've broken...

### CODE REVIEW

**Comment by larsoner** [comment]
Failures trickling in already :)

**Comment by larsoner** [comment]
You'll need to `import pytest` *inside* the test function. It's a quirk of the notebook tests...

**Comment by drammock** [comment]
> You'll need to `import pytest` _inside_ the test function. It's a quirk of the notebook tests...

thanks for the tip

File: mne/bem.py [hunk 1]
```diff
@@ -1953,13 +1890,8 @@ def make_flash_bem(subject, overwrite=False, show=True, subjects_dir=None,
...
     mri_dir = Path(mri_dir)
     bem_dir = Path(bem_dir)
-    if flash_path is None:
-        flash_path = mri_dir / 'flash' / 'parameter_maps'
-    else:
-        warn("The flash_path parameter is deprecated and will be removed "
-             "in version 1.1. Use the flash5_img parameter instead.")
-        flash_path = Path(flash_path).resolve()
     subjects_dir = env['SUBJECTS_DIR']
+    flash_path = (mri_dir / 'flash' / 'parameter_maps').resolve()
```
Comment (larsoner) [inline mne/bem.py:1894]: I think the `resolve` will fail if `makedirs` hasn't been called yet, so you need
```suggestion
    flash_path = (mri_dir / 'flash' / 'parameter_maps').resolve(strict=False)
```
https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve

**Review by larsoner — COMMENTED** [review]
I like the removal of modified_env, too! Other than one problem LGTM +1 for merge

File: mne/bem.py (hunk 1 above)
Comment (drammock) [reply mne/bem.py:1894]: `strict=False` is the default

**Comment by drammock** [comment]
All green! @agramfort you want to do the honors? This touches your recent BEM changes.

**Review by agramfort — APPROVED** [review]
@hoechenberger can you maybe just check it does not break the pipeline?

**Comment by hoechenberger** [comment]
@agramfort Yes, can do later tonight

**Comment by drammock** [comment]
@hoechenberger did you have a chance to check this against mne-bids-pipeline yet?

**Comment by hoechenberger** [comment]
@drammock I will do it tonight CEST! :)

**Comment by drammock** [comment]
@hoechenberger what is the status here?  I'd like the deprecations to happen soon, so that users on `main` have as much time as possible to be exposed to them before the new release.

**Comment by hoechenberger** [comment]
@drammock one of the changes broke autoreject, but otherwise all was good
@agramfort had been working on a fix but I don't know the current status of this

I'd say it's safe to merge this though, as autoreject was the only problematic package that popped up

**Comment by drammock** [comment]
looks like the fix is in autoreject already: https://github.com/autoreject/autoreject/pull/284

**Comment by hoechenberger** [comment]
Thanks Dan!!

**Pull request merged**